src/Compilers/Core/CodeAnalysisTest/CodeAnalysisTest.csproj

```diff
@@ -74,6 +74,10 @@
       <Project>{F7712928-1175-47B3-8819-EE086753DEE2}</Project>
       <Name>TestUtilities.FX45</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Test\Utilities\CSharp\CSharpCompilerTestUtilities.csproj">
+      <Project>{4371944A-D3BA-4B5B-8285-82E5FFC6D1F9}</Project>
+      <Name>CSharpCompilerTestUtilities</Name>
+    </ProjectReference>
```

Comment:
Is this dependency necessary for ```TestOptions.RegularWithIOperationFeature```? If so, could that `CSharpParseOptions` instance be duplicated in this assembly rather than an adding a `ProjectReference`?